pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,117 @@
+# Test ownership
+Test ownership is critical to adopting a healthy testing culture at your organization. By defining test owners, your teams will become accountable for maintaining a fast and reliable test suite, ensuring confidence when you deploy your code.
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
+Test ownership is managed via team assignments in a TESTOWNER file. The team that is the default owner of a test [will be automatically assigned flaky tests](/docs/test-analytics/flaky-test-assignment) to triage.
+
+> 🚧 Buildkite test ownership is currently in private beta
+> Please reach out to our support team to register for early access.
+
+## Setting test ownership
+
+You can upload a TESTOWNER file via this API endpoint:
+
+```bash
+curl --location 'https://analytics-api.buildkite.com/v1/test-ownerships' \
+     --header "Authorization: Bearer <your-suite-api-token>" \
+     -F 'file=@<your-TESTOWNER-file-location>'
+```
+You might consider creating a new pipeline to automatically upload your TESTOWNER file when changes are detected.
+
+## Example TESTOWNER file
+
+```bash
+# This is a comment.
+# Each line is a file pattern followed by one or more team slugs.
+
+# These teams will be the owners for everything in
+# the repo. While both teams own the test, the first team listed will
+# become the default owner for tests in this directory. Unless
+# a later match takes precedence, team-slug-1 will have any
+# flaky tests assigned to them by default.
+*       team-slug-1 team-slug-2
+
+# Order is important; the last matching pattern takes the most
+# precedence. Any test file ending with _spec.rb will be assigned
+# to the test-analytics team and not team-slug-1.
+*_spec.rb    test-analytics #This is an inline comment.
```

Comment:
Minor tweaks ... please correct if my interpretation is incorrect ...

```suggestion
# Any test file ending with _spec.rb will be assigned
# to the test-analytics team and not team-slug-1.
# The last matching team takes precedence.
*_spec.rb test-analytics #This is an inline comment.
```

A couple of comments/questions:

1. Removed extra space (as per my previous comment).
1. Can multiple teams be specified against this file and if so, what's the behaviour in doing so?

Reply:
The suggested change isn't quite right; we might have touched on this in slack already, but to clarify:

> Can multiple teams be specified against this file and if so, what's the behaviour in doing so?

Yes, multiple teams can own the same test location. The team listed first will be the "default owner" of the test, meaning when a test flakes they alone will be auto-assigned to fix it. There's more detail in the section above starting with `# These teams will be the owners for everything in`

> Is this configuration meant to override the * team-name-style configurations above?

Correct, the order in this entire file is important. If our file was just:
```bash
*              team-slug-1 team-slug-2
*_spec.rb      test-analytics
```
We assign ownership of all tests matching *_spec.rb` to the `test-analytics` team. Then, excluding tests matching `*_spec.rb`, we assign all tests (`*`) to `team-slug-1` and `team-slug-2`.

Reply:
Thanks! Fixed and reworded following a more detailed discussion with you :-)